package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "ni-system-link-server-grafana-plugins",
+  "version": "1.0.0",
+  "description": "SystemLink Server plugins for Grafana",
+  "scripts": {},
+  "author": "Carson Moore",
+  "license": "Apache-2.0",
+  "devDependencies": {},
+  "engines": {
+    "node": ">=18"
+  },
+  "dependencies": {}
```

The bare minimum needed for a dummy package is just the name and version, if we do need to keep it:
```suggestion
  "description": "Placeholder package for SystemLink Server plugins for Grafana",
  "private": true
```

---

Done. See comment above re: name for additional context.